scripts/check_format.sh

```diff
@@ -30,7 +30,7 @@ function check_shogun_style {
         echo "-----"
         echo "Running clang-format-3.8 against branch ${2:-}, with hash $BASE_COMMIT"
 
-        COMMIT_FILES=$(git diff --name-only $BASE_COMMIT)
+        COMMIT_FILES=$(git diff --name-only $BASE_COMMIT src/shogun)
```

Comment:
Hi @OXPHOS
The PR #3934 should solve the issue. Once it gets merged, please rebase this PR on develop and remove the commit https://github.com/shogun-toolbox/shogun/pull/3932/commits/2fc8ab7b002981794bd13e80c4d6d3e3c2b5b020. After that, everything should become green on CIs.